### PR TITLE
Install glusterfs client from stretch-backports

### PR DESCRIPTION
**Pull request by moelsayed**
https://github.com/rancher/rancher/issues/22802

### CODE REVIEW

**Comment by superseb** [comment]
Resulting in the following installs:

```
Reading state information...                                                                                                                                                                                                                                                                                                  
The following additional packages will be installed:                                                                                                                                                                                                                                                                          
  python-jwt python-prettytable                                                                                                                                                                                                                                                                                               
Suggested packages:                                                                                                                                                                                                                                                                                                           
  python-crypto                                                                                                                                                                                                                                                                                                               
Recommended packages:                                                                                                                                                                                                                                                                                                         
  python-cryptography                                                                                                                                                                                                                                                                                                         
The following NEW packages will be installed:                                                                                                                                                                                                                                                                                 
  python-jwt python-prettytable                                                                                                                                                                                                                                                                                               
The following packages will be upgraded:                                                                                                                                                                                                                                                                                      
  glusterfs-client glusterfs-common                                                                                                                                                                                                                                                                                           
2 upgraded, 2 newly installed, 0 to remove and 41 not upgraded.                                                                                                                                                                                                                                                               
Need to get 8052 kB of archives.                                                                                                                                                                                                                                                                                              
After this operation, 230 kB disk space will be freed.                                                                                                                                                                                                                                                                        
Get:1 http://deb.debian.org/debian stretch/main amd64 python-prettytable all 0.7.2-3 [22.4 kB]                                                                                                                                                                                                                                
Get:2 http://deb.debian.org/debian stretch/main amd64 python-jwt all 1.4.2-1+deb9u1 [15.3 kB]                                                                                                                                                                                                                                 
Get:3 http://deb.debian.org/debian stretch-backports/main amd64 glusterfs-common amd64 4.1.5-1~bpo9+1 [5654 kB]                                                                                                                                                                                                               
Get:4 http://deb.debian.org/debian stretch-backports/main amd64 glusterfs-client amd64 4.1.5-1~bpo9+1 [2361 kB]                                                                                                                                                                                                               
debconf: delaying package configuration, since apt-utils is not installed                                                                                                                                                                                                                                                     
Fetched 8052 kB in 0s (28.0 MB/s)    
```

File: Dockerfile [hunk 1]
```diff
@@ -9,4 +9,9 @@ RUN sed -i -e 's!\bmain\b!main contrib!g' /etc/apt/sources.list && \
     && clean-install \
     xfsprogs \
     open-iscsi \
-    azure-cli \
+    azure-cli
+
+RUN echo "deb http://deb.debian.org/debian stretch-backports main" >> /etc/apt/sources.list.d/backports.list &&\
```
Comment (superseb) [inline Dockerfile]: Is this purposely done on a new line? Adding the repo and using `clean-install -t stretch-backports $package` should also work.

If it needs to be on a separate line, please keep the same format as above:

```
$VAR1 \
&& $VAR2
```

Or change all to the new style.

File: Dockerfile [hunk 2]
```diff
@@ -9,4 +9,9 @@ RUN sed -i -e 's!\bmain\b!main contrib!g' /etc/apt/sources.list && \
...
+    apt-get update && \
+    apt-get -t stretch-backports install glusterfs-client glusterfs-common -y && \
```
Comment (superseb) [inline Dockerfile]: Is there any policy on backports that prevents breaking update if we don't pin this version?

Comment (moelsayed) [reply Dockerfile]: Not that I know of.

**Comment by superseb** [comment]
LGTM

**Pull request merged**